pkg/pillar/go.mod

```diff
@@ -109,6 +110,7 @@ require (
 	github.com/morikuni/aec v1.0.0 // indirect
 	github.com/opencontainers/runc v1.1.3 // indirect
 	github.com/opencontainers/selinux v1.10.1 // indirect
+	github.com/peterbourgon/g2s v0.0.0-20170223122336-d4e7ad98afea // indirect
```

Comment:
github says: This repository has been archived by the owner on Feb 14, 2019. It is now read-only.
Is there some replacement or alternative which is being maintained?

Reply:
sure, it is used for simple send over UDP, I will add it in the collector package and will remove the dependency.

Reply:
removed the dependency to g2s.